test/code_size/hello_webgl_wasm2js.json

```diff
@@ -1,8 +1,8 @@
 {
   "a.html": 346,
   "a.html.gz": 262,
-  "a.js": 21690,
-  "a.js.gz": 11405,
-  "total": 22036,
-  "total_gz": 11667
+  "a.js": 21720,
```

Comment:
slightly odd size increases here. do you know why?

Reply:
I guess because the program previously only used HEAP32 but now also uses HEAP8?